docs/develop/api_for_vms.rst

```diff
@@ -0,0 +1,46 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
```

Comment:
Please include the description to the the three fields - Event Name, Start Date and Venue.